src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div class="product" data-qa="card">
+      <img
+        class="product__image"
+        src="images/imac.jpeg"
+        alt="Monoblock"
+      >
+      <h2 class="product__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h2>
+      <div class="product__code">
+        Код товара: 195434
+      </div>
```

Comment:
```suggestion
      <span class="product__code">
        Код товара: 195434
      </span>
```
'span' here would be more semantic